modules/light-clients/08-wasm/types/expected_interfaces.go

```diff
@@ -0,0 +1,83 @@
+package types
+
+import (
+	wasmvm "github.com/CosmWasm/wasmvm"
+	wasmvmtypes "github.com/CosmWasm/wasmvm/types"
+)
+
```

Comment:
should we add a type assertion that the wasmvm implements this interface?

Reply:
sure thing!